website/docs/docs/deploy/webhooks.md

```diff
@@ -167,20 +167,21 @@ An example of a webhook payload for an errored run:
 You can use the dbt Cloud API to create new webhooks that you want to subscribe to, get detailed information about your webhooks, and to manage the webhooks that are associated with your account. The following sections describe the API endpoints you can use for this. 
 
 :::info Access URLs
-dbt Cloud is hosted in multiple regions in the world and each region has a different access URL. People on Enterprise plans can choose to have their account hosted in any one of these regions. This section uses `cloud.getdbt.com` (which is for North America) as part of the endpoint but your access URL might be different. For a complete list of available dbt Cloud access URLs, refer to [Regions & IP addresses](/docs/cloud/about-cloud/regions-ip-addresses).   
+dbt Cloud is hosted in multiple regions in the world and each region has a different access URL. People on Enterprise plans can choose to have their account hosted in any one of these regions. For a complete list of available dbt Cloud access URLs, refer to [Regions & IP addresses](/docs/cloud/about-cloud/regions-ip-addresses).   
 :::
 
 ### List all webhook subscriptions
 List all webhooks that are available from a specific dbt Cloud account.
 
 #### Request 
 ```shell
-GET https://cloud.getdbt.com/api/v3/accounts/{account_id}/webhooks/subscriptions
+GET https://{your access URL}/api/v3/accounts/{account_id}/webhooks/subscriptions
 ```
 
 #### Path parameters
 | Name       | Description                          |
 |------------|--------------------------------------|
+| `your access URL` | The login URL for your dbt Cloud account. |
```

Comment:
piggybacking on what I said above. we should have a dedicated page that explains these concepts:

- what is a login URL
- what is an access URL

these might be different, and we should take care to say what they are, how they're constructed, how to find them, and when to use each one

Reply:
@matthewshaver I think this is an improvement as-is, but maybe I can help write the "connect" page and then come back and update these docs with the language we decide on? idk. let me know how I can help

Reply:
@cmcarthur Is this a requiremnent for the initial client(s) we're enabling? I have it on my to-do list but was going to work on it the second week in Sept. I don't mind writing it up!